tests/unit_tests/test_wave_02.py

```diff
@@ -34,7 +34,11 @@ def test_get_no_matching_items_by_category():
 
     items = vendor.get_by_category("electronics")
 
-    raise Exception("Complete this test according to comments below.")
     # *********************************************************************
     # ****** Complete Assert Portion of this test **********
     # *********************************************************************
+    assert len(items)== 0
+    assert item_a not in items
+    assert item_b not in items
+    assert item_c not in items
```

Comment:
Since we know from the first assert that `items` is empty, we do not need to check if these items are in there because we already know they are not!